File: src/main/java/io/github/wntopia/gikipedia/server/domain/auth/dto/internal/DataGsmSession.java
package io.github.wntopia.gikipedia.server.domain.auth.dto.internal;

import io.github.wntopia.gikipedia.server.domain.auth.dto.response.DataGsmUserInfoResDto;
import java.io.Serializable;
import java.time.Instant;

public record DataGsmSession(
    DataGsmUserInfoResDto userInfo, DataGsmToken token, Instant authenticatedAt)
    implements Serializable {}

File: src/main/kotlin/io/github/wntopia/gikipedia/server/domain/auth/dto/internal/DataGsmSession.kt
package io.github.wntopia.gikipedia.server.domain.auth.dto.internal

import io.github.wntopia.gikipedia.server.domain.auth.dto.response.DataGsmUserInfoResDto
import java.io.Serializable
import java.time.Instant

data class DataGsmSession(
    val userInfo: DataGsmUserInfoResDto,
    val token: DataGsmToken,
    val authenticatedAt: Instant,
) : Serializable
